amd_openvx_extensions/amd_rpp/include/vx_ext_rpp.h

```diff
@@ -982,9 +982,11 @@ extern "C"
 	/*! \brief [Graph] Creates a RPP Brightness function node.
 	 * \ingroup group_amd_rpp
 	 * \param [in] graph The handle to the graph.
-	 * \param [in] pSrc The input tensor data.
+	 * \param [in] pSrc The input tensor in <tt>\ref VX_TYPE_UINT8<tt> or <tt>\ref VX_TYPE_FLOAT32<tt> or 
+	 * <tt>\ref VX_TYPE_FLOAT16<tt> or <tt>\ref VX_TYPE_INT8<tt> format data.
```

Comment:
@fiona-gladwin why the extra line *. This might cause issues with Doxygen. Keep all comments for on parameter as one continuous line

Reply:
Done. changed into single line